app/services/billing_invoice/fill.rb

```diff
@@ -4,6 +4,8 @@ module BillingInvoice
   class Fill < ApplicationService
     parameter :invoice, required: true
 
+    Error = Class.new(ApplicationService::Error)
```

Comment:
This class has been introduced because when we use the `BillingInvoice::Fill` service, we expect it to raise an error of type `BillingInvoice::Fill::Error`. However, currently, the service raises an error of type `ApplicationService::Error`, which is not the expected behavior. When I want to catch service king of error class I'll use `ApplicationService::Error` explicitly.